aspnetcore/performance/caching/distributed.md

```diff
@@ -129,10 +129,10 @@ To configure NCache:
 
 1. Install [NCache open source NuGet](https://www.nuget.org/packages/Alachisoft.NCache.OpenSource.SDK/).
 1. Configure the cache cluster in [client.ncconf](https://www.alachisoft.com/resources/docs/ncache/admin-guide/client-config.html).
-1. Add the following code to `Startup.ConfigureServices`:
+1. Add the following code to `Program.cs`:
 
    ```csharp
-   services.AddNCacheDistributedCache(configuration =>    
+   builder.Services.AddNCacheDistributedCache(configuration =>    
```

Comment:
Move this snippet to `Program.cs`. Be sure to test the program.

Reply:
done .